### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by KingsleyIbe**
# In this milestone, I implemented the following:

- Removed all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component for the calculator project design

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const Calculator = () => (
```
Comment (rloterh) [inline src/Components/Calculator.js]: - [ ] Your _Calculator_ component _(Calculator.js)_ is not classed based. Kindly change this component to a class-based component.

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
-import './App.css';
+import './styles/App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
```
Comment (rloterh) [inline src/App.js:4]: - [ ] Your _App_ component _(App.js)_ is not classed based. Kindly change this component to a class-based component.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kingsley, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hi Robert,

Thank you for your feedback.
Changes will be implemented

**Review by AshakaE — APPROVED** [review]
### Hi @KingsleyIbe   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hi Ashaka,

Thanks for your prompt response and approval

**Pull request merged**